pkg/services/cloudmigration/cloudmigrationimpl/cloudmigration.go

```diff
@@ -678,3 +726,25 @@ func (s *Service) CancelSnapshot(ctx context.Context, sessionUid string, snapsho
 
 	return nil
 }
+
+func (s *Service) getLocalEventId(ctx context.Context) string {
+	anonId, ok, err := s.kvStore.Get(ctx, "anonymous_id")
+	if err != nil {
+		s.log.Error("Failed to get usage stats id", "error", err)
+		return ""
```

Comment:
Is the intent to send a event even if a local event id is not present?

Reply:
good call, will add an error to the return type